sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/Client/CryptographyClient.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Azure.Security.KeyVault.Cryptography.Client
+{
+    using Azure.Core;
+    using Azure.Core.Pipeline;
+    using Azure.Security.KeyVault.Cryptography.Base;
+    using Azure.Security.KeyVault.Cryptography.CryptoProvider;
+    using Azure.Security.KeyVault.Cryptography.Interface;
+    using Azure.Security.KeyVault.Cryptography.Utilities;
+    using Azure.Security.KeyVault.Keys;
+    using System;
+
+    /// <summary>
+    /// The Cryptography Client 
+    /// </summary>
+    public class CryptographyClient
+    {
+        #region const
+        const string DEFAULT_KV_SCOPE_URI = @"https://vault.azure.net/.default";
+        #endregion
+
+        #region fields
+
+        #endregion
+
+        #region Properties
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public TokenCredential Credentials { get; }
+        /// <summary>
+        /// Cryptography Client Options
+        /// </summary>
+        public CryptographyClientOptions CryptoClientOptions { get; }
+
+        /// <summary>
+        /// HttpPipeline
+        /// </summary>
+        public HttpPipeline Pipeline {get;}
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public ICryptographyProvider CryptoProvider { get; protected set; }
+
+        /// <summary>
+        /// KeyVault Uri
+        /// </summary>
+        public Uri KeyVaultKeyUri { get; }
+
+        /// <summary>
+        /// KeyVault Key
+        /// </summary>
+        public Key KeyVaultKey { get; }
+        #endregion
+
+        #region Constructor
+        /// <summary>
+        /// Initializes a new instance of Cryptography Client
+        /// </summary>
+        protected CryptographyClient()
+        {
+            KeyVaultKeyUri = null;
+            KeyVaultKey = null;
+        }
+
+        #region Local Crypto
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="keyVaultKeyUri"></param>
+        public CryptographyClient(Uri keyVaultKeyUri)
```

Comment:
keyVaultKeyUri -> keyId

Reply:
Also the constructors which don't take credential should be removed, you need a credential to instantiate the client

Reply:
The constructor that has no credentials are meant for local crypto?
Or am I missing anything?